src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandlerTests.cs

```diff
@@ -23,6 +23,7 @@ public class MigrateRepoCommandHandlerTests
     private const string ADO_REPO = "foo-repo";
     private const string GITHUB_ORG = "foo-gh-org";
     private const string GITHUB_REPO = "gh-repo";
+    private const string TARGET_API_URL = "https://api.github.com";
```

Comment:
_I will add a comment just for this line but it applies to all tests for all CLIs._ 

There is no need to add the target api url here because it's only going to be used inside the `Command` (not the handler) to create an instance of `GithubApi`. Instead a new and very simple test needs to be added to the Command tests (e.g. `MigrateOrgCommandTests.cs`) to make sure if the `--target-api-url` is provided, the factory is going to receive it.

Reply:
The new tests are going to be similar to [this one](https://github.com/github/gh-gei/blob/c5c54d27fe71ec8c4b40c83fca996e1697050d73/src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepo/MigrateRepoCommandTests.cs#L54)